addon/components/hyper-table/index.hbs

```diff
@@ -1,125 +1,129 @@
-      @icon="fa fa-columns"
-      @label={{t "hypertable.features.manage_fields.title"}}
-      class="margin-bottom-px-6 fields-manager-toggle icon-responsive"
-      data-control-name="table_toggle_fields"
-      {{action "openAvailableFields" bubbles=false}} />
+        @skin="primary"
+        @icon="fa fa-columns"
+        @label={{t "hypertable.features.manage_fields.title"}}
+        class="margin-bottom-px-6 fields-manager-toggle icon-responsive"
+        data-control-name="table_toggle_fields"
+        {{action "openAvailableFields" bubbles=false}} />
 
-    <HyperTable::Views
-      @manager={{this.manager}} @closeAvailableViews={{action "closeAvailableViews"}} />
+      <HyperTable::Views
+        @manager={{this.manager}} @closeAvailableViews={{action "closeAvailableViews"}} />
 
-    <div
-      class="available-fields-wrapper {{if this.manager.availableFieldsPanel 'visible' 'invisible'}}"
-      {{on-click-outside (action "closeAvailableFields")}}>
-      <div class="available-fields-wrapper__categories">
-        <div
-          class="field-category {{unless this._activeFieldCategory 'field-category--active'}}"
-          {{action "setFieldCategory"}}>
-          <div>{{t "hypertable.features.manage_fields.all_fields"}}</div>
-          <div><i class="fa fa-caret-right"></i></div>
-        </div>
-        {{#each this._fieldCategories as |category|}}
+      <div
+        class="available-fields-wrapper {{if this.manager.availableFieldsPanel 'visible' 'invisible'}}"
+        {{on-click-outside (action "closeAvailableFields")}}>
+        <div class="available-fields-wrapper__categories">
           <div
-            class="field-category {{if (eq this._activeFieldCategory category.key) 'field-category--active'}}"
-            data-control-name={{concat "field_category_toggle_" category.key}}
-            {{action "setFieldCategory" category.key}}>
-            <div>{{category.label}}</div>
+            class="field-category {{unless this._activeFieldCategory 'field-category--active'}}"
+            {{action "setFieldCategory"}}>
+            <div>{{t "hypertable.features.manage_fields.all_fields"}}</div>
             <div><i class="fa fa-caret-right"></i></div>
           </div>
-        {{/each}}
-      </div>
-
-      <div class="available-fields-wrapper__fields">
-        <div class="search">
-          <Input
-            @type="text" @value={{this._availableFieldsKeyword}}
-            data-control-name="field_search_input" class="upf-input"
-            placeholder={{t "hypertable.features.manage_fields.search_placeholder"}}
-          />
+          {{#each this._fieldCategories as |category|}}
+            <div
+              class="field-category {{if (eq this._activeFieldCategory category.key) 'field-category--active'}}"
+              data-control-name={{concat "field_category_toggle_" category.key}}
+              {{action "setFieldCategory" category.key}}>
+              <div>{{category.label}}</div>
```

Comment:
this.category.label or @category.label ?

Reply:
Neither of two, it comes from the following code `{{#each this._fieldCategories as |category|}}`